### ISSUE TITLE
[MD057] Relative link '...' does not exist errors since v0.192

### ISSUE THREAD

**Issue by Ecno92** [rvben/rumdl#190]
`uv tool run rumdl==0.0.191 check .` reports `Success: No issues found in 26 files (55ms)` in our project.

While 

`uv tool run rumdl==0.0.192 check .` reports

```
...
docs/index.md:12:30: [MD057] Relative link 'bli/index.md' does not exist
docs/index.md:13:24: [MD057] Relative link 'bla/index.md' does not exist
...
```

The links itself are like this:

```markdown
- [Guide](bli/index.md)
```

The `index.md` file is in the bli directory.

The problem is also present for relative files in the same directory.

**Comment by rvben** [comment]
Hi @Ecno92, thanks a lot for raising this. I'll investigate right away.

**Comment by rvben** [comment]
Clear bug that slipped through the testing. A variable was set once, so that tests succeeded, but when multiple files were linted the bug showed up. This is now fixed in 0.0.193.

Thanks again for raising this issue and for making rumdl better by doing so! 🚀
